academic_helper/templatetags/custom_tags.py

```diff
@@ -34,3 +37,33 @@ def raw_include(name: str):
     with open(path) as file:
         output = file.read()
     return mark_safe(output)
+
+
+@register.simple_tag(takes_context=True)
+def rating_for_comment(context, num):
+    request = context.get('request')
+    if request is None:
+        raise Exception('Make sure you have "django.core.context_processors.request" in your templates context processor list')
```

Comment:
Nice.
`num` should be `score` I guess?
I would not pass this function just the score, but the entire rating object.
Alternatively, I would call it `rating_for_score` (as it currently has nothing to do with a comment).

Reply:
1. num -> score: Agree - in the next PR.
2. rating_for_comment-> rating_for_score: Disagree, rating_for_score has no meaning (rating=score). this is a score that is shown next to a comment. maybe simple_rating()?
3. pass the entire rating object: Disagree, a function should accept the minimum possible input.